### ISSUE TITLE
[chr4:134389657-134389854][INS][SVLEN=197bp]

### ISSUE THREAD

**Issue by jzook** [jzook/testHG008curation#176]
`chr4	134389657	Minda_106	N	<INS>	.	PASS	SVLEN=197;SVTYPE=INS;SUPP_VEC=ONT_severus_INS18039,ONT_Sniffles2.INS.394M3,ONT_ID_18348_1,ONT_i_282,PB_severus_INS15907,PB_Sniffles2.INS.37FM3,PB_i_675,PB_ID_10215_1
`

https://v2.genomeribbon.com/?session=https://42basepairs.com/download/s3/giab-data/ribbon-json/ribbon-hg008-cov-bedpe.json&locus=chr4:134389657#ribbon

**Comment by jzook** [comment]
Looks like a true insertion but the size seems to be closer to the 226bp that DRAGEN has. Unsure of the exact length because it has a long homopolymer and seems to be a type of LINE insertion
<img width="985" alt="Screenshot 2024-10-01 at 2 18 34 PM" src="https://github.com/user-attachments/assets/29dcdcfd-e6ab-45ae-903b-a3fb58cd6f4f">

<img width="1669" alt="Screenshot 2024-10-01 at 2 30 49 PM" src="https://github.com/user-attachments/assets/10cd0d13-1121-47a1-adf6-b9616ef18100">

**Comment by jo-mc** [comment]
yes a truncated Line1 insertion of about 223bp, and including a 19 bp target site duplication (TSD).

Inserted sequence: TAAAAAAATTTAAAATTTTTGGTGGGGTCGGGGGAGGGGGGAGGGATAGCATTGGGAGATATACCTAATGCTAGATGACACATTAGTGGGTGCAGTGCACCAGCATGGCACATGTATACATATGTAACTAACCTGCACAATGTGCACATGTACCCTAAAACTTAGAGTATAATAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

TSD: AAAAAAATTTAAAATTTTT

Concordance to L1HS:  19S198M6S   with 4bp mismatch.

**Comment by jo-mc** [comment]
comment/note: Looked up why we seeing LINE-1 (and why short Line-1 with 5' inversions, which I thought were rare):

 **_Transposable elements in cancer (pancreatic section)_**
https://www.nature.com/articles/nrc.2017.35 
In contrast, in pancreatic ductal adenocarcinoma (PDAC), most fully developed malignancies (∼90%) express LINE-1 ORF1p. This loss of retroelement control is not seen with the same uniformity early in the disease; only 11% of the histological precursor lesions, pancreatic intraepithelial neoplasias (PanINs), express LINE-1 ORF1p[127](https://www.nature.com/articles/nrc.2017.35#ref-CR127). Although ORF1p expression levels vary between individuals, there is little heterogeneity within a single tumour or between different sites of disease in the same patient[132](https://www.nature.com/articles/nrc.2017.35#ref-CR132).

**_Retrotransposon insertions in the clonal evolution of pancreatic ductal adenocarcinoma_**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4775273/
Pancreatic ductal adenocarcinoma (PDAC) is typically diagnosed after the disease has metastasized; it is among the most lethal forms of cancer. We recently described aberrant expression of a protein, ORF1p, encoded by Long INterspersed Element (LINE-1) retrotransposon, in PDAC [1](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4775273/#R1). To test whether LINE-1 expression leads to somatic insertions of this mobile DNA, we used a targeted method to sequence LINE-1 insertion sites in matched PDAC and normal samples. **We found evidence of 465 somatic LINE-1 insertions in 20 PDAC genomes absent from corresponding normal samples.** In cases where matched normal tissue, primary PDAC, and metastatic disease sites were available, insertions were found in primary and metastatic in differing proportions. Two adenocarcinomas secondarily involving the pancreas but originating in the stomach and duodenum acquired insertions with a similar discordance between primary and metastatic sites. Together, our findings show that LINE-1 contributes to the genetic evolution of PDAC and suggest that somatic insertions are acquired discontinuously in gastrointestinal neoplasms. [Inverted or reversed orientation of the 5′ L1Hs segment was commonly seen, detail in supp data on 5' inversion (twin priming)]
